### PR TITLE
Feat/auto focus title

### PR DESCRIPTION
**Pull request by z0rzi**

### CODE REVIEW

File: src/app/views/Editor/EditorPane.vue [hunk 1]
```diff
@@ -148,6 +149,16 @@ export default class EditorPane extends Vue {
...
+  @Watch('readonly', { immediate: true })
+  private onReadOnlyChange () {
+    if (!this.readonly) {
+      this.storyscriptSLS = new StoryscriptSLS()
+    } else {
+      this.storyscriptSLS = null
+    }
+  }
+
+  @Watch('readonly')
```
Comment (JeanBarriere) [inline src/app/views/Editor/EditorPane.vue]: these are unrelated to the PR. Can you revert this file changes ? :)

File: src/internal/components/Inputs/EditableText.vue [hunk 2]
```diff
@@ -58,5 +58,11 @@ export default class SEditableText extends Vue {
   private cleanString (str: string): string {
     return str.trim().replace(/\\n/g, '')
   }
+
+  public focus () {
+    const input = this.$refs.self as any
```
Comment (JeanBarriere) [inline src/internal/components/Inputs/EditableText.vue]: any will do, but if we can avoid it by putting a HTMLElement or HTMLInputElement, would be great

File: src/app/views/Editor/EditorPane.vue (hunk 1 above)
Comment (z0rzi) [reply src/app/views/Editor/EditorPane.vue]: Oops, I kind of mixed up my git branches, my bad :no_mouth:

I created a new PR with the requested changes:
https://github.com/storyscript/studio/pull/420
